charts/loki-stack/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: "v1"
 name: loki-stack
-version: 2.6.2
-appVersion: v2.1.0
+version: 2.6.3
```

```suggestion
version: 2.6.4
```

Needs a version bump

---

@gcaracuel done